### PR TITLE
Input components

### PR DESCRIPTION
**Pull request by utkarshducoder**
Please review my input components

this page contains the input components of Component library

live link: https://componentsofcss.netlify.app/component%20html/input.html

Following are the features of this page

nav: contains brand name and links to the home page and components library.
main: contains the documentation of input components with snippets and examples.
aside: contains links to all my component libraries.
footer: contains links to my social media

image

image

### CODE REVIEW

File: utkarsh component/input/input.css [hunk 1]
```diff
@@ -0,0 +1,755 @@
...
+font-family: 'Roboto', sans-serif;
+font-family: 'Satisfy', cursive;
+color: var(--secondary-color);
+}
+
+.current-pg{
+    color: var(--secondary-color);
+    display: block;
+        background-color: #eaf1dd;
+        /* color: whitesmoke; */
```
Comment (khushijohri2001) [inline utkarsh component/input/input.css]: Remove the unwanted code here. To make the code looks clean and readable

File: utkarsh component/input/input.css [hunk 2]
```diff
@@ -0,0 +1,755 @@
...
+        text-decoration: none;
+        font-family: Verdana, Geneva, sans-serif;
+        font-size: 1.5em;
+        padding: 6px 4px;
+        margin: 4px;
+        border-right: 2px solid #999999;
```
Comment (khushijohri2001) [inline utkarsh component/input/input.css:71]: you can use a var here for the color

**Review by khushijohri2001 — APPROVED** [review]

File: utkarsh component/input/input.css [hunk 3]
```diff
@@ -0,0 +1,755 @@
...
+
+.current-pg{
+    color: var(--secondary-color);
+    display: block;
+        background-color: #eaf1dd;
+        /* color: whitesmoke; */
+        text-decoration: none;
+        font-family: Verdana, Geneva, sans-serif;
+        font-size: 1.5em;
+        padding: 6px 4px;
```
Comment (i-am-a-codegirl) [inline utkarsh component/input/input.css:69]: avoid using absolute units

File: utkarsh component/input/input.css [hunk 4]
```diff
@@ -0,0 +1,755 @@
...
+a:hover{
+    color: #f472b6;
+  
+}
+
+main{
+    
+    /* grid-template-areas: 200px 1fr; */
+    /* align-items: center; 
+    padding: 3.5rem;
```
Comment (i-am-a-codegirl) [inline utkarsh component/input/input.css]: remove unwanted code

File: utkarsh component/input/input.css [hunk 5]
```diff
@@ -0,0 +1,755 @@
...
+}
+#check{
+  display: none;
+}
+@media (max-width: 925px){
+  label.logo{
+    font-size: 30px;
+    padding-left: 50px;
+  }
+  nav  li a{
```
Comment (i-am-a-codegirl) [inline utkarsh component/input/input.css]: put media queries in the end

**Review by i-am-a-codegirl — APPROVED** [review]

**Pull request merged**